.github/workflows/check-backport.yml

```diff
@@ -0,0 +1,50 @@
+name: Stable Backport Check
+on:
+  issue_comment:
+  pull_request_target:
+    types:
+      - labeled
+
+jobs:
+  check-backport:
+    name: Check Backport
+    if: ${{ ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') ) || github.event.label.name == 'stable backport' }}
```

Comment:
You can simplify this check a bit by using this https://github.com/marketplace/actions/detect-an-issue-comment-on-a-pull-request

I'm currently not too sure this check skips properly comments made on issues and not pull requests.

Reply:
I just copied the existing `rebase` one for this to keep them consistent. I'd rather not futz any more with it now that it works.